resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -147,6 +147,8 @@ lambda.region.tooltip=The AWS Region to use for the connected AWS account.
 lambda.sam.buildInContainer.tooltip=Builds the serverless application's functions inside an AWS Lambda-like Docker container locally before deployment.
 lambda.sam.skipCheckingNewerContainerImage.tooltip=Skips pulling down the latest Docker image for the specified AWS Lambda runtime.
 lambda.sam.dockerNetwork.tooltip=The name or ID of an existing Docker network that the AWS Lambda Docker containers should connect to, along with the default bridge network.
+lambda.sam.additionalBuildArgs.tooltip=Additional args to add to the 'sam build' command, see https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-cli-command-reference-sam-build.html
+lambda.sam.additionalLocalArgs.tooltip=Additional args to add to the 'sam local' command, see https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-cli-command-reference-sam-local-invoke.html
```

Comment:
These go to the 'sam local invoke' command, also I'm not sure theres much point including the doc link since it's probably not clickable through the UI.
![Screen Shot 2019-08-20 at 10 31 44 AM](https://user-images.githubusercontent.com/4661536/63369894-d2460d00-c335-11e9-91ad-d20f206149d3.png)